code/modules/clothing/spacesuits/rig/suits/ert.dm

```diff
@@ -8,6 +8,7 @@
 	suit_type = "ERT commander"
 	icon_state = "ert_commander_rig"
 	emp_protection = 35
+	slowdown = 1
```

Comment:
This is such a terrible way to implement it. First of all you don't need lines 38, 57 and 74 as line 11 make all ert rigs have slowdown of 1 even HAPT as well, which was as you mentioned not supposed to.

Please add this override to the rig definition and add `slowdown = 0` override inside of the HAPT definition